apps/deploy-web/src/components/remote-deploy/github/Framework.tsx

```diff
@@ -0,0 +1,123 @@
+import { useState } from "react";
+import { Select, SelectContent, SelectGroup, SelectItem, SelectTrigger, SelectValue, Spinner } from "@akashnetwork/ui/components";
+import clsx from "clsx";
+import { Globe } from "iconoir-react";
+import { useAtom } from "jotai";
+
+import remoteDeployStore from "@src/store/remoteDeployStore";
+import { ServiceType } from "@src/types";
+import { usePackageJson } from "../api/api";
+import { useBitPackageJson } from "../api/bitbucket-api";
+import { useGitlabPackageJson } from "../api/gitlab-api";
+import { removeInitialUrl } from "../utils";
+const frameworks = [
```

Comment:
We have that array duplicated in the `FrameworkDetection` component. Please extract the array in a utils.